registry/storage/blobwriter_resumable.go

```diff
@@ -0,0 +1,9 @@
+// +build !noresumabledigest
```

Comment:
The issue is a forked crypto library for sha256/sha512 that allows resumption of hashes. Under certain conditions, certain organizations cannot ship it. Adding this build flag will disable resumable digest.